cmd/airflow.go

```diff
@@ -328,6 +338,64 @@ func newAirflowBashCmd() *cobra.Command {
 	return cmd
 }
 
+func newAirflowObjectRootCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "object",
+		Aliases: []string{"obj"},
+		Short:   "Manage local Airflow Connections, Variables, and Pools",
+		Long:    "Manage local Airflow Connections, Variables, and Pools. You can export and import this objects from a local Airflow environment to an Airflow settings file",
+	}
+	cmd.AddCommand(
+		newObjectImportCmd(),
+		newObjectExportCmd(),
+	)
+	return cmd
+}
+
+func newObjectImportCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "import",
+		Short: "Create and update local Airflow objects from an Aiflow Settings file",
+		Long:  "This command will create all connections, variables, and pools in an Airflow Settings file locally. Airflow must be running locally for this command to work",
+		// ignore PersistentPreRunE of root command
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+			return nil
+		},
+		PreRunE: utils.EnsureProjectDir,
+		RunE:    airflowSettingsImport,
+	}
+	cmd.Flags().BoolVarP(&connections, "connections", "c", false, "Import connections from an Airflow Settings File")
+	cmd.Flags().BoolVarP(&variables, "variables", "v", false, "Import variables from an Airflow Settings File")
+	cmd.Flags().BoolVarP(&pools, "pools", "p", false, "Import pools from an Airflow Settings File")
+	cmd.Flags().StringVarP(&settingsFile, "settings-file", "s", "airflow_settings.yaml", "Settings or env file export objects too")
+	cmd.Flags().BoolVarP(&logs, "verbosity", "l", false, "Show logs from airflow commands used to import objects")
```

Comment:
There is already a global `--verbosity` flag, can't we simply enable that for all our commands irrespective of Software/Cloud. It would mean an overall more consistent UX.